scripts/build/package.js

```diff
@@ -94,19 +94,26 @@ data.icons.forEach((icon) => {
   icons.push(icon);
 
   const iconObject = iconToObject(icon);
+  const iconExportName = slugToVariableName(icon.slug);
 
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  writeJs(jsFilepath, `module.exports=${iconObject};`);
+  const newImportMessage = `use "const { ${iconExportName} } = require('simple-icons/icons')" instead`;
```

Comment:
Another stupid but newbie-friendly change:

```suggestion
  const newImportMessage = `use "const { ${iconExportName} } = require('simple-icons/icons');" instead`;
```